src/ui/windows/TogglDesktop/TogglDesktop/ui/controls/TimelineTimeEntryBlock.xaml.cs

```diff
@@ -7,9 +9,41 @@ namespace TogglDesktop
     /// </summary>
     public partial class TimelineTimeEntryBlock : UserControl
     {
+        public TimeEntryBlock ViewModel
+        {
+            get => DataContext as TimeEntryBlock;
+            set => DataContext = value;
+        }
+
         public TimelineTimeEntryBlock()
         {
             InitializeComponent();
+
+        }
+
+        private void OnThumbBottomDragDelta(object sender, DragDeltaEventArgs e)
+        {
+            if (ViewModel.Height + e.VerticalChange > 0)
+                ViewModel.Height += e.VerticalChange;
+        }
+
+        private void OnThumbTopDragDelta(object sender, DragDeltaEventArgs e)
+        {
+            if (ViewModel.Height + e.VerticalChange > 0)

```

Comment:
Something is wrong with this method - you're checking whether Height + VerticalChange > 0, but then you're subtracting the VerticalChange.

Reply:
You're right, thank you.